__init__.py

```diff
@@ -378,6 +424,9 @@ def _init_setup_options(self):
         if "stt_blacklist" not in self.settings:
             self.settings["stt_blacklist"] = ["ovos-stt-plugin-pocketsphinx"]
 
+        if "single_tts_list" not in self.settings:
+            self.settings["single_tts_list"] = False
```

Comment:
default this to True please, this new list view is a bigscreen specific requirement and does not match the official OPM api, at a later point we may decide to make this a default but for now we need to keep compat

Reply:
This is not bigscreen specific this is for manjaro ovos images, any other image can decide how they want their tts menu workflow to look